Mathlib/Algebra/Group/Defs.lean

```diff
@@ -39,6 +39,7 @@ instance One.ofOfNat1 {α} [OfNat α (nat_lit 1)] : One α where
 class Inv (α : Type u) where
   inv : α → α
 
+/-- `x⁻¹` is the multiplicative inverse of `x`. -/
```

Comment:
See my comment about the change in nolints.json: https://github.com/leanprover-community/mathlib4/pull/386/files#r954269146

Reply:
Not sure if `inheritDoc` is available on the current version of lean used by mathlib4, but if it is, the preferred solution here is to mark the syntax as `@[inheritDoc]` and put the doc comment on `Inv.inv` instead.

Reply:
Done. I think https://github.com/leanprover-community/mathlib4/pull/374 made `inheritDoc` available.

Reply:
I made the documentation a bit more generic, because I'm not sure what happens with `to_additive`.